docs/guides/connect-aepp-to-wallet.md

```diff
@@ -1,90 +1,109 @@
+    onAddressChange: async () => {
+      this.publicAddress = await this.aeSdk.address()
+      this.balance = await this.aeSdk.balance(this.publicAddress)
     },
     onDisconnect: () => {
-      this.resetState()
+      // you may want to reset state here
       alert('Disconnected')
     }
   })
-  this.height = await this.aeSdk.height()
-  // start looking for wallets, see step 2.
   this.scanForWallets()
-  // open iframe with Wallet if run in top window
-  if (window === window.parent) this.openReverseIframe()
 }
 ```
 
-## 2. Scan for wallets
+## 3. Scan for wallets
 ```js
-scanForWallets () {
-  const handleWallets = async function ({ wallets, newWallet }) {
-    newWallet = newWallet || Object.values(wallets)[0]
-    if (confirm(`Do you want to connect to wallet ${newWallet.name}`)) {
-      this.detector.stopScan()
-      // connect to the wallet, see step 3.
-      await this.connectToWallet(newWallet)
+methods: {
+  async scanForWallets()  {
+    const handleWallets = async function ({ wallets, newWallet }) {
+      newWallet = newWallet || Object.values(wallets)[0]
+      if (confirm(`Do you want to connect to wallet ${newWallet.name}`)) {
+        detector.stopScan()
+        // connect to the wallet, see step 4.
+        await this.connectToWallet(newWallet)
+      }
     }
-  }
 
-  const scannerConnection = BrowserWindowMessageConnection({
-    connectionInfo: { id: 'spy' }
-  })
-  this.detector = WalletDetector({ connection: scannerConnection })
-  this.detector.scan(handleWallets.bind(this))
+    const scannerConnection = BrowserWindowMessageConnection({
+      connectionInfo: { id: 'spy' }
+    })
+
+    const detector = await WalletDetector({ connection: scannerConnection })
+    detector.scan(handleWallets.bind(this))
+  }
 }
 ```
 
-## 3. Connect to Wallet
+## 4. Connect to a wallet
+
+Append method for wallet connection
 
 ```js
-async connectToWallet (wallet) {
+async connectToWallet(wallet) {
   await this.aeSdk.connectToWallet(await wallet.getConnection())
-  this.accounts = await this.aeSdk.subscribeAddress('subscribe', 'connected')
-  this.pub = await this.aeSdk.address()
-  this.onAccount = this.pub
-  this.balance = await this.aeSdk.getBalance(this.pub)
-  this.walletName = this.aeSdk.rpcClient.info.name
-  this.addressResponse = await this.aeSdk.address()
-  this.heightResponse = await this.aeSdk.height()
-  this.nodeInfoResponse = await this.aeSdk.getNodeInfo()
+  this.connectedAccounts = await this.aeSdk.subscribeAddress('subscribe', 'connected')
+  this.publicAddress = await this.aeSdk.address()
+  this.balance = await this.aeSdk.getBalance(this.publicAddress)
+  this.node = await this.aeSdk.getNodeInfo()
 }
 ```
 
 ## 4. Use Wallet's Node for chain communication
 
-AEPP can request the wallet to share its connected node URLs if any to interact with the chain.
+Aepp can request the wallet to share its connected node URLs if any to interact with the chain.
```

Comment:
Actually it is not a step at all, we need to decide should we use it by default or not cc: @subhod-i